bowtie.wdl

```diff
@@ -0,0 +1,43 @@
+task Bowtie_se {
```

Comment:
`bowtie.wdl` will be imported as `bowtie` by default. Which means these tasks are called as `bowtie.Bowtie_se`.
I propose that you change this to just `task se {` the task can then be called as `bowtie.se`